### ISSUE TITLE
ab-av1 overrides -svtav1-params set with --enc

### ISSUE THREAD

**Issue by veikk0** [alexheretic/ab-av1#128]
I was testing tune=1 vs tune=0 with the latest ab-av1 git and ended up with identical output files (matching sha1sums). Looking at the actual FFmpeg command being run, it seems that since ab-av1 is adding its own `-svtav1-params scd=0`, any -svtav1-params the user specifies with --enc is overridden (with multiples of a parameter, only the latter is applied by FFmpeg).

Since SvtAv1EncApp usage was removed, I had basically forgotten about the `--svt` parameter, which seems to be the way to set -svtav1-params currently. I've been using `--enc` for all the encoders I use with ab-av1, but due to the above, the behavior isn't consistent between SVT-AV1 and other encoders.

`scd` [hasn't done anything](https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/1929) since SVT-AV1 0.9.1, so it has no effect in recent SVT-AV1 versions.

Command to reproduce:

>ab-av1 auto-encode -e libsvtav1 --enc svtav1-params=tune=0:keyint=300 --min-vmaf 93.4 --pix-format yuv420p --preset 8 -o tune0.ivf --input input.mp4

**Comment by mr44er** [comment]
Yes, you need to use it multiple times: `... --svt tune=0 --svt scd=0 ...`
`--enc af=loudnorm` as example is meant for ffmpeg


> the behavior isn't consistent between SVT-AV1 and other encoders
That is because libsvtav1 is the default used encoder. Use `ab-av1 auto-encode --help` to see the defaults.

**Comment by alexheretic** [comment]
Thanks for raising this, it's mostly setup to use `--svt` for these args. I agree that with your usage the behaviour is surprising.

We could:
* Make user specified `--enc svtav1-params=...` override any default params.
* Fail if using this **and** `--svt`.

Or alternatively just ban `svtav1-params` usage, which is simpler. This might be better since it nudges users to the easier to use API.

Also note, since you used svtav1 "keyint" param, `--keyint` arg handles this (setting ffmpeg -g) and defaults to 10s.

Regarding `--scd` it probably is worth removing this as a first class option & default behaviour.

**Comment by veikk0** [comment]
I'd prefer being able to use --enc svtav1-params for consistency between encoders and easier scripting. IMO svtav1-params is easier to remember once you're using multiple encoders and setting various encoder parameters with the -{encoder}-params switch. It also makes scripting easier, since FFmpeg has kept the naming of the switch consistent between encoders.

**Comment by alexheretic** [comment]
I guess the first solution would satisfy that?

> * Make user specified --enc svtav1-params=... override any default params.
> * Fail if using this and --svt.

**Comment by veikk0** [comment]
Yes, it would.
